Bio/Nexus/Nexus.py

```diff
@@ -185,7 +185,7 @@ def __init__(self, symbols, gap):
                 self.set(x, y, 0)
 
     def set(self, x, y, value):
-        """Swaps the value."""
+        """Swap the value."""
```

Comment:
This docstring was not very helpful, do you understand what this really does?

Reply:
Not quite, since the positioning inside the `data` dictionary doesn't really change if the `x` and `y` are swapped..
I decided to remove the docstring. Only the `__init__` and `smprint` methods hold one.